addons/project/static/src/components/project_status_with_color_selection/project_status_with_color_selection_field.js

```diff
@@ -20,11 +20,22 @@ export class ProjectStatusWithColorSelectionField extends SelectionField {
         return this.colors[value] ? this.colorPrefix + this.colors[value] : "";
     }
 }
+
+ProjectStatusWithColorSelectionField.props = {
+    ...SelectionField.props,
+    statusLabel: { type: String, optional: true },
+};
+
 ProjectStatusWithColorSelectionField.template = 'project.ProjectStatusWithColorSelectionField';
 
 export const projectStatusWithColorSelectionField = {
     ...selectionField,
     component: ProjectStatusWithColorSelectionField,
+    extractProps: ({ attrs }) => {
+        return {
+            statusLabel: attrs.status_label,
+        };
+    },
```

Comment:
I did not see but there are also an `extractProps` function in `selectionField`. And so we should do this:
```suggestion
    extractProps: (fieldInfo) => {
        const props = selectionField.extractProps(fieldInfo);
        props.statusLabel = fieldInfo.attrs.status_label;
        return props;
    },
```

Reply:
Your suggestion didn't work so I made some changes to your suggestion.okay for you ?

       extractProps({ options }) {
          const props = selectionField.extractProps(...arguments);
          props.statusLabel = options.status_label;
          return props;
      }

Reply:
Oh it is because I forgot to pass dynamicLabel in parameter for the parent function. 

If it does not also work then yes we can do as you suggest.